roles/post_install/openstack_tasks/tasks/main.yml

```diff
@@ -30,6 +30,20 @@
   import_tasks: instances.yml
   when: instance and resource_state == 'present'
 
+- name: Networks
+  tags: network
+  import_tasks: networks
```

Comment:
Should this be `networks_and_routers.yml`?

Reply:
Thanks ! Fixed